.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: cpp
+language: c
 compiler:
   - gcc
   - clang
 
 # This is a lie: we don't need sudo but this is required to get an
 # Ubuntu image with a libc that isn't ancient, and with cmocka libs.
```

Comment:
Yeah that comment was obsolete as soon as travis started supporting a docker & trusty environment. When we started using travis the trusty instance was VMs only. I'll strip this out before pushing. Good catch by the way :+1: